dev-utilities/committer-utils/metron-committer-common

```diff
@@ -135,8 +135,8 @@ function read_pull_request {
       exit 1
     fi
 
-    if [[ "$PR_MERGEABLE" != "\"clean\"" ]]; then
-      echo "Error: pull request #$PR is not mergable"
+    if [[ "$PR_MERGEABLE" == "\"clean\"" ]] || [[ "$PR_MERGEABLE" == "unstable" ]]; then
```

Comment:
I think you meant `if [[ "$PR_MERGEABLE" != "\"clean\"" ]] || [[ "$PR_MERGEABLE" == "unstable" ]]; then`

Reply:
drat